backend/alembic/versions/9c615d167588_add_tasks.py

```diff
@@ -0,0 +1,79 @@
+"""Add Tasks
+
+Revision ID: 9c615d167588
+Revises: 617bf951f6a2
+Create Date: 2018-07-14 19:07:16.544507
+
+"""
+from alembic import op
+import sqlalchemy as sa
+
+# revision identifiers, used by Alembic.
+revision = '9c615d167588'
+down_revision = '3f2c98c1710b'
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    op.create_table('Tasks',
+                    sa.Column('_created', sa.DateTime(), server_default=sa.text('now()'), nullable=False),
+                    sa.Column('_modified', sa.DateTime(), server_default=sa.text('now()'), nullable=False),
+                    sa.Column('id', sa.Integer(), autoincrement=True, nullable=False),
+                    sa.Column('key', sa.String(length=50), nullable=False),
+                    sa.Column('name', sa.String(length=100), nullable=False),
+                    sa.Column('image_path', sa.String(length=100), nullable=False),
+                    sa.PrimaryKeyConstraint('id', name=op.f('pk_Tasks')),
+                    sa.UniqueConstraint('key', name=op.f('uq_Tasks_key'))
+                    )
+    op.create_table('ScanCategories_Tasks',
+                    sa.Column('scan_category_id', sa.Integer(), nullable=False),
+                    sa.Column('task_id', sa.Integer(), nullable=False),
+                    sa.ForeignKeyConstraint(['scan_category_id'], ['ScanCategories.id'],
+                                            name=op.f('fk_ScanCategories_Tasks_scan_category_id_ScanCategories')),
+                    sa.ForeignKeyConstraint(['task_id'], ['Tasks.id'],
+                                            name=op.f('fk_ScanCategories_Tasks_task_id_Tasks'))
+                    )
+    op.add_column('LabelTags', sa.Column('task_id', sa.Integer(), nullable=False))
```

Comment:
I couldn't run this migration due to below error:

```
Traceback (most recent call last):
  File "/Users/jakubpowierza/Documents/MedTagger/backend/venv/bin/alembic", line 11, in <module>
    sys.exit(main())
  File "/Users/jakubpowierza/Documents/MedTagger/backend/venv/lib/python3.6/site-packages/alembic/config.py", line 486, in main
    CommandLine(prog=prog).main(argv=argv)
  File "/Users/jakubpowierza/Documents/MedTagger/backend/venv/lib/python3.6/site-packages/alembic/config.py", line 480, in main
    self.run_cmd(cfg, options)
  File "/Users/jakubpowierza/Documents/MedTagger/backend/venv/lib/python3.6/site-packages/alembic/config.py", line 463, in run_cmd
    **dict((k, getattr(options, k, None)) for k in kwarg)
  File "/Users/jakubpowierza/Documents/MedTagger/backend/venv/lib/python3.6/site-packages/alembic/command.py", line 254, in upgrade
    script.run_env()
  File "/Users/jakubpowierza/Documents/MedTagger/backend/venv/lib/python3.6/site-packages/alembic/script/base.py", line 427, in run_env
    util.load_python_file(self.dir, 'env.py')
  File "/Users/jakubpowierza/Documents/MedTagger/backend/venv/lib/python3.6/site-packages/alembic/util/pyfiles.py", line 81, in load_python_file
    module = load_module_py(module_id, path)
  File "/Users/jakubpowierza/Documents/MedTagger/backend/venv/lib/python3.6/site-packages/alembic/util/compat.py", line 82, in load_module_py
    spec.loader.exec_module(module)
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "alembic/env.py", line 69, in <module>
    run_migrations_online()
  File "alembic/env.py", line 63, in run_migrations_online
    context.run_migrations()
  File "<string>", line 8, in run_migrations
  File "/Users/jakubpowierza/Documents/MedTagger/backend/venv/lib/python3.6/site-packages/alembic/runtime/environment.py", line 836, in run_migrations
    self.get_context().run_migrations(**kw)
  File "/Users/jakubpowierza/Documents/MedTagger/backend/venv/lib/python3.6/site-packages/alembic/runtime/migration.py", line 330, in run_migrations
    step.migration_fn(**kw)
  File "/Users/jakubpowierza/Documents/MedTagger/backend/alembic/versions/9c615d167588_add_tasks.py", line 37, in upgrade
    op.add_column('LabelTags', sa.Column('task_id', sa.Integer(), nullable=False))
  File "<string>", line 8, in add_column
  File "<string>", line 3, in add_column
  File "/Users/jakubpowierza/Documents/MedTagger/backend/venv/lib/python3.6/site-packages/alembic/operations/ops.py", line 1565, in add_column
    return operations.invoke(op)
  File "/Users/jakubpowierza/Documents/MedTagger/backend/venv/lib/python3.6/site-packages/alembic/operations/base.py", line 319, in invoke
    return fn(self, operation)
  File "/Users/jakubpowierza/Documents/MedTagger/backend/venv/lib/python3.6/site-packages/alembic/operations/toimpl.py", line 123, in add_column
    schema=schema
  File "/Users/jakubpowierza/Documents/MedTagger/backend/venv/lib/python3.6/site-packages/alembic/ddl/impl.py", line 169, in add_column
    self._exec(base.AddColumn(table_name, column, schema=schema))
  File "/Users/jakubpowierza/Documents/MedTagger/backend/venv/lib/python3.6/site-packages/alembic/ddl/impl.py", line 115, in _exec
    return conn.execute(construct, *multiparams, **params)
  File "/Users/jakubpowierza/Documents/MedTagger/backend/venv/lib/python3.6/site-packages/sqlalchemy/engine/base.py", line 948, in execute
    return meth(self, multiparams, params)
  File "/Users/jakubpowierza/Documents/MedTagger/backend/venv/lib/python3.6/site-packages/sqlalchemy/sql/ddl.py", line 68, in _execute_on_connection
    return connection._execute_ddl(self, multiparams, params)
  File "/Users/jakubpowierza/Documents/MedTagger/backend/venv/lib/python3.6/site-packages/sqlalchemy/engine/base.py", line 1009, in _execute_ddl
    compiled
  File "/Users/jakubpowierza/Documents/MedTagger/backend/venv/lib/python3.6/site-packages/sqlalchemy/engine/base.py", line 1200, in _execute_context
    context)
  File "/Users/jakubpowierza/Documents/MedTagger/backend/venv/lib/python3.6/site-packages/sqlalchemy/engine/base.py", line 1413, in _handle_dbapi_exception
    exc_info
  File "/Users/jakubpowierza/Documents/MedTagger/backend/venv/lib/python3.6/site-packages/sqlalchemy/util/compat.py", line 265, in raise_from_cause
    reraise(type(exception), exception, tb=exc_tb, cause=cause)
  File "/Users/jakubpowierza/Documents/MedTagger/backend/venv/lib/python3.6/site-packages/sqlalchemy/util/compat.py", line 248, in reraise
    raise value.with_traceback(tb)
  File "/Users/jakubpowierza/Documents/MedTagger/backend/venv/lib/python3.6/site-packages/sqlalchemy/engine/base.py", line 1193, in _execute_context
    context)
  File "/Users/jakubpowierza/Documents/MedTagger/backend/venv/lib/python3.6/site-packages/sqlalchemy/engine/default.py", line 509, in do_execute
    cursor.execute(statement, parameters)
sqlalchemy.exc.IntegrityError: (psycopg2.IntegrityError) column "task_id" contains null values
 [SQL: 'ALTER TABLE "LabelTags" ADD COLUMN task_id INTEGER NOT NULL'] (Background on this error at: http://sqlalche.me/e/gkpj)
```

**Reproduction:**
1. Checkout to the master branch.
2. Clean your setup (remove all data and DB structures).
3. Run MedTagger on master with all default fixtures and etc.
4. Add some Labels.
5. Stop MedTagger.
6. Checkout to your branch.
7. Run your DB migration.
8. Boom!

This way you can reproduce environment that will be running on someone's dev/prod instance :)

**Idea for future:** Maybe we should prepare tests for migrations? Hmm... Quite nice!